src/ui/components/entityregions/EntityRegions.js

```diff
@@ -16,10 +16,11 @@ const EntityRegions = (props) => {
     const history = useHistory();
     const location = useLocation();
     const searchParams = queryString.parse(location.search);
-
+    const visibleRegionParams = searchParams.visible ? searchParams.visible : "";
     React.useEffect(() => {
         const defaultVisibility = (region) => regionMaximized === region.id ||
-            (!isHiddenRegion(region.id) && (region.customVisibility ? region.customVisibility() : true));
+            visibleRegionParams.includes(region.id) ||
```

Comment:
This will fail for the following example:
- Imagine we are seeing the WorkOrder page (https://github.com/cern-eam/eam-light-frontend/blob/master-cern/src/ui/pages/work/Workorder.js).
- Let's say we want to have just the `CUSTOMFIELDSEQP` and `COMMENTS` regions visible. For that, we pass `...?visible=CUSTOMFIELDSEQP,COMMENTS`. 
- But we will end up seeing an unwanted 3rd region because we have another region id that matches this condition: `CUSTOMFIELDS` (https://github.com/cern-eam/eam-light-frontend/blob/master-cern/src/ui/pages/work/Workorder.js#L331)